types/google-protobuf/index.d.ts

```diff
@@ -196,14 +196,14 @@ export class Map<K, V> {
     fieldNumber: number,
     writer: BinaryWriter,
     keyWriterFn: (field: number, key: K) => void,
-    valueWriterFn: (field: number, value: V, writeCallback?: BinaryWriteCallback) => void,
+    valueWriterFn: (field: number, value: V, writerCallback: BinaryWriteCallback) => void,
     writeCallback?: BinaryWriteCallback
   ): void;
   static deserializeBinary<K, V>(
     map: Map<K, V>,
     reader: BinaryReader,
     keyReaderFn: (reader: BinaryReader) => K,
-    valueReaderFn: (reader: BinaryReader, defaultValue?: V, readCallback?: BinaryReadCallback) => V,
+    valueReaderFn: (reader: BinaryReader, value: any, readerCallback: BinaryReadCallback) => V,
```

Comment:
Without knowing anything about this API, I would have guessed that "defaultValue" was the correct name, since it seems odd to pass a value to a value reader.